src/test/java/uk/nhs/prm/e2etests/test/repositorye2etest/RepositoryNackTests.java

```diff
@@ -47,6 +56,7 @@ class RepositoryNackTests {
     private final EhrTransferServiceLargeEhrOQ ehrTransferServiceLargeEhrOQ;
     private final EhrTransferServiceNegativeAcknowledgementOQ ehrTransferServiceNegativeAcknowledgementOQ;
     private final EhrTransferServiceParsingDeadLetterQueue ehrTransferServiceParsingDeadLetterQueue;
+    private final NhsProperties nhsProperties;
```

Comment:
This is now only used by the commented out NACK 99 test, could we please remove these lines until we reintroduce the 99 NACK test?